### PR TITLE
adding `sort -u` to ${repos} and removing exit code check

### PR DESCRIPTION
**Pull request by missingcharacter**
`if` in [bash already evaluates if the return status is 0](http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html).
`sort -u` will take care of possible repeated repos when trying to mirror multiple image versions.

### CODE REVIEW

File: mirror/fileRepository/mirror.sh [hunk 1]
```diff
@@ -33,30 +33,25 @@ mirror_kubeupv2(){
...
 
-    for i in nodeup utils.tar.gz kops
-    do
-    for platform in linux darwin
+    for file in nodeup utils.tar.gz
     do
-        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/$i -O $TMPDIR/$platform/amd64/$i
-        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/$i.sha1 -O $TMPDIR/$platform/amd64/$i.sha1
-    done
+        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/linux/amd64/$file -O $TMPDIR/linux/amd64/$file
```
Comment (pahud) [inline mirror/fileRepository/mirror.sh]: We still tend to mirror platform for mac os x especially the`kops` client

File: mirror/mirror-images.sh [hunk 2]
```diff
@@ -113,7 +112,7 @@ function pull_and_push(){
 # list all existing repos
 all_ecr_repos=$(aws --profile=bjs --region $ECR_REGION ecr describe-repositories --query 'repositories[*].repositoryName' --output text)
 echo "$all_ecr_repos"
-repos=$(grep -v ^# $IMAGES_FILE_LIST | cut -d: -f1)
+repos=$(grep -v ^# $IMAGES_FILE_LIST | cut -d: -f1 | sort -u)
```
Comment (pahud) [inline mirror/mirror-images.sh]: appreciated!

File: mirror/fileRepository/mirror.sh (hunk 1 above)
Comment (missingcharacter) [reply mirror/fileRepository/mirror.sh]: I noticed that is why [I moved it up](https://github.com/nwcdlabs/kops-cn/blob/e7c72cde2a449da950108d465e9c67b3ac61eb6f/mirror/fileRepository/mirror.sh#L36-L37)

```
        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/kops -O $TMPDIR/$platform/amd64/kops
        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/kops.sha1 -O $TMPDIR/$platform/amd64/kops.sha1
```

**Comment by pahud** [comment]
LGTM
thanks for the PR! @missingcharacter

**Pull request merged**